README.md

```diff
@@ -331,27 +325,38 @@ Here are the command global flags that can be used for every command:
 
 > #### Usage
 >
->     certifaction prepare [prepare flags] [-o output] [input | url]
+>   certifaction prepare [input files | URLs...] [flags]
 >
 > #### Description
 >
-> Prepares a document for signing. A salt and a document claim encryption key
-> pair is added to the document, and optionally a branded footer. If enabled, a
-> Digital Twin QR code with a document encryption key will be added to the
-> document. If the input file is already a digital original then the command
-> will return the input file unchanged. Currently only processes PDF files. An
-> error is returned if the input is not a PDF file. If the input parameter is
-> omitted, then the command will take its input from stdin. If the output
-> parameter is omitted, then the output will be returned to stdout.
->
+> Prepare a document for signing.
+> A salt and a document claim encryption key pair is added to the document, and optionally a branded footer.
+> If enabled, a Digital Twin QR code with a document encryption key will be added to the document.
+> If the input file is already a digital original then the command will return the input file unchanged.
+> Currently only processes PDF files. An error is returned if the input is not a PDF file.
+> If the input file (or URL) is omitted, then the command will take its input from stdin.
+> If the output file is omitted, then the output will be returned to stdout.
+> 
+> The signature and the Digital Twin QR code can be placed manually or on an additional page; they are placed on an additional page by default.
+> To place them manually, set --additional-page=false and pass the necessary positioning flags (see --additional-page help).
+> 
 > #### Flags
 >
->     --digital-twin     Adds a Digital Twin footer, encrypts the document and stores it.
->                       Uses the footer template given by the -footer parameter.
->                       If no -footer parameter is provided, then the default footer is used.
->     --language         Overrides the default language. Available languages are de, en, fr and it.
->     --scope            Optional signature scope override to choose between
->                       register, sign and certify.
+>       --additional-page                Adds an additional page with the signatures of the signers and the Digital Twin QR code.
+>                                        If --additional-page=false, --page, --position-x, --position-y, and --height need to be submitted;
```

Comment:
--page, --position-x, --position-y and --height are not described below.

Reply:
That omission has been there, the position flags were not added to the prepare command in the CLI. I'll fix it later.

Reply:
Now I think we shouldn't waste effort on duplicating CLI help output here. Any helpful information should be added in the CLI help.